tests/unit/client.py

```diff
@@ -493,7 +547,8 @@ def test_port_connect_nic_error(self):
         C.port.register('mock-01', 'gi1/1/6')
         C.port.connect_nic('mock-01', 'gi1/1/6', 'node-09', 'eth0')
         with pytest.raises(FailedAPICallException):
-            C.port.connect_nic('mock-01', 'gi1/1/6', 'node-08', 'eth0')
+            C.port.connect_nic('mock-01', 'gi1/1/6', 'node-09', 'eth0')
```

Comment:
Why change this here? What the original test seems to do is connect a port (gi1/1/6 on mock-01), that's already connected to a nic (eth0 on node-**09**), to another nic on a different node (eth0 on node-**08**). That would still raise the error correctly.